cmd/pinniped-proxy/src/cli.rs

```diff
@@ -12,8 +12,16 @@ pub struct Options {
     #[structopt(
         short = "p",
         long = "port",
+        env = "PINNIPED_PROXY_PORT",
         default_value = "3333",
         help = "Specify the port on which pinniped-proxy listens."
     )]
-    pub port: u16, 
+    pub port: u16,
+    #[structopt(
+        long = "default-ca-cert",
+        env = "PINNIPED_PROXY_DEFAULT_CA_CERT",
+        default_value = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
```

Comment:
Using the normal location by default, but enabling configuration for local use (or custom k8s setups).